src/Uno.UI.Runtime.Skia.Gtk/GtkHost.cs

```diff
@@ -172,7 +172,7 @@ void CreateApp(ApplicationInitializationCallbackParams _)
 				app.Host = this;
 			}
 
-			WUX.Application.Start(CreateApp, _args);
+			WUX.Application.StartWithArguments(CreateApp);
```

Comment:
I'm feeling something is wrong here. The field `_args` is only assigned but not used. Note that the constructor of GtkHost is public. So consumers instantiating `GtkHost` will find their passed `args` value being ignored.

One important thing here is: "Is there any use case where external consumers instantiate `GtkHost`?" If there is no such use case, I think a breaking change should be considered.

Reply:
Yes, the fact that it is public is a problem. I would expect that nobody is actually changing the `_args` manually, but an option is to expose it as a public property on the host for example. Removing completely would be a big breaking change so that sounds like something for 5.0... 

What do you think @jeromelaban?

Reply:
This could indeed be a problem. I'd suggest to pass the arguments always, and compare the joined values with the value return by `Environment.GetCommandLineArgs`. If it's the same, then we can use `Environment.GetCommandLineArgs` as the base value but with properly separated arguments?

Reply:
@jeromelaban it is unfortunately not possible to compare the joined values, as .NET already performs some processing there so it removes quotes around items which have them, so the "raw" command line will most likely look different, and it is also not easily possible to transform the arguments array back to a raw string as that would require adding quotes smartly - but would still mean that the Arguments the user gets from `LaunchActivatedEventArgs` will not really match what was the actual command line input. I would vote for ignoring the input array, because if anyone actually wants to use it and access it in code, they can still do it (by storing it in some static variable for example). In most scenarios the user will not touch the arguments manually and in such cases they will get the same thing in the raw string anyway.

Reply:
Ok, let's do this then. Thanks!